tracing/tracers/lightstep/lightstep.go

```diff
@@ -99,6 +100,10 @@ func parseOptions(opts []string) (lightstep.Options, error) {
 		case "cmd-line":
 			cmdLine = parts[1]
 			logCmdLine = true
+		case "protocol":
+			if parts[1] != "grpc" {
+				useGRPC = false
```

Comment:
I'd make that explicit `http` or `grpc` with `grpc` as default. It's easier to figure out what you get instead of relying on the default in the tracer (that used to be `grpc` instead of `http` in earlier versions IIRC).

Reply:
I agree, let's make it explicit http, anticipating other variants, or subvariants of the current ones in the same option

Reply:
makes sense, but it has the drawback of changing defaults.